ROMFS/px4fmu_common/init.d/airframes/4071_ifo

```diff
@@ -121,3 +117,7 @@ then
 	param set MAV_1_MODE 0	# normal
 	param set SER_TEL2_BAUD 57600
 fi
+
+set MIXER quad_x
+set PWM_OUT 1234
```

Comment:
As a relevant note and in reviewing my diff here, `4071_ifo` does not call `rc.mc_defaults`, as is also the case with `4072_draco`.